data/moves.js

```diff
@@ -2341,9 +2334,8 @@ let BattleMovedex = {
 			},
 		},
 		isZ: "kommoniumz",
-		secondary: {
-			// Sheer Force negates the selfBoost even though it is not secondary
-		},
+		// @ts-ignore ~ Sheer Force negates the selfBoost even though it is not secondary
+		secondary: {},
```

Comment:
Why does this need a `ts-ignore`? `secondary` should be allowed to be an empty object.